modules/rhcos-enabling-multipath-day-2.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="rhcos-enabling-multipath-day-2_{context}"]
+= Enabling multipathing with kernel arguments on {op-system}
+
+{op-system} supports multipathing on the primary disk, allowing stronger resilience to hardware failure to achieve higher host availability. Post-installation support is available by activating multipathing via the machine config.
+
+[IMPORTANT]
+====
+Enabling multipathing during installation is supported and recommended for nodes provisioned in {product-title} 4.8 or higher. In setups where any I/O to non-optimized paths results in I/O system errors, you must enable multipathing at installation time. For more information about enabling multipathing during installation time, see "Enabling multipathing with kernel arguments on RHCOS" in the _Installing on bare metal_ documentation.
+====
+
+[IMPORTANT]
+====
+On IBM Z and LinuxONE, you can enable multipathing only if you configured your cluster for it during installation. For more information, see "Installing {op-system} and starting the {product-title} bootstrap process" in _Installing a cluster with z/VM on IBM Z and LinuxONE_.
+====
+// Add xref once it's allowed.
+
+.Prerequisites
+* You have a running {product-title} cluster that uses version 4.7 or later.
+* You are logged in to the cluster as a user with administrative privileges.
+
+.Procedure
+
+. To enable multipathing post-installation on control plane nodes (also known as the master nodes):
```

Comment:
Is there a steadfast rule on parentheses? I've always been told to steer away from them.

Reply:
Good catch. With 4.7, I believe we were using this boilerplate wording that was using parentheses. I've gone ahead and removed it here because we are no longer using the "aka" wording starting with 4.8. (And yes, you're right, we try to follow ISG's suggestion to avoid parentheses by rewriting or by using commas instead. 😁 )